schema/filter_artworks.js

```diff
@@ -60,21 +63,39 @@ export const FilterArtworksCounts = {
 
 export const FilterArtworksType = new GraphQLObjectType({
   name: "FilterArtworks",
+  interfaces: [NodeInterface],
+  isTypeOf: obj => has(obj, "hits") && has(obj, "aggregations"),
   fields: () => ({
+    __id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: "The ID of the object.",
+      resolve: ({ options }) => {
+        return toGlobalId("FilterArtworks", JSON.stringify(options))
+      },
```

Comment:
so `toGlobalId` is a deterministic uuid generator that takes the JS and the name filter artworks?

Can be switched to 

```js
     resolve: ({ options }) =>  toGlobalId("FilterArtworks", JSON.stringify(options)),
```

if you want